pkg/eventcounter/api/api.go

```diff
@@ -89,6 +93,7 @@ func NewEventCounterService(
 		featureClient:                f,
 		accountClient:                a,
 		druidQuerier:                 d,
+		eventStorage:                 v2ecstorage.NewEventStorage(b, bigQueryDataset, l),
```

Comment:
Can we call this function in cmd/server/server.go such as kafka? If we do that, we don't have to assign bigQueryDataset in `NewEventCounterService` 👍 

https://github.com/bucketeer-io/bucketeer/blob/main/pkg/eventpersister/cmd/server/server.go#L346

Reply:
Although we can call it from cmd/server/server.go, that makes a dependency from `eventcounter/cmd/server` pkg to `eventcounter/storage/v2` pkg.

I don't want to create dependency from `{module}/cmd/server` to `{module}/storage` as possible.

Reply:
> I don't want to create dependency from {module}/cmd/server to {module}/storage as possible.
I thought the same thing, but it could confuse other developers because the way to write the code is different. How about adding comments about those things?